### PR TITLE
Multiple address for different networks on creation and edition of projects

### PR DESCRIPTION
**Pull request by mateodaza**
Requires this [PR](https://github.com/Giveth/impact-graph/pull/531) to be on staging 
Both must be deployed at the same time

### CODE REVIEW

File: src/components/views/join/JoinSubscriptionCard.tsx [hunk 1]
```diff
@@ -86,7 +86,6 @@ const JoinSubscriptionCard = () => {
 								disabled={!validateEmail(email)}
 								label='Subscribe'
 								buttonType='primary'
-								type='submit'
```
Comment (MohammadPCh) [inline src/components/views/join/JoinSubscriptionCard.tsx:89]: We need to set the button type.

File: src/components/views/homepage/HomeGetUpdates.tsx [hunk 2]
```diff
@@ -85,7 +85,6 @@ const HomeGetUpdates = () => {
 							<SubscribeButton
 								disabled={!validateEmail(email)}
 								label='SUBSCRIBE'
-								type='submit'
```
Comment (MohammadPCh) [inline src/components/views/homepage/HomeGetUpdates.tsx:88]: We need to set the button type.

File: src/config/development.ts [hunk 3]
```diff
@@ -9,6 +9,8 @@ import { gwei2wei } from '@/helpers/blockchain';
 
 const INFURA_API_KEY = process.env.NEXT_PUBLIC_INFURA_API_KEY;
 const BASE_ROUTE = 'https://serve.giveth.io';
+// IMPORTANT: Using dev to make it work until staging is merged
+// const BASE_ROUTE = 'https://dev.serve.giveth.io';
```
Comment (MohammadPCh) [inline src/config/development.ts:13]: Can I test it on staging or it's still on the dev branch?

File: src/components/views/donate/CryptoDonation.tsx [hunk 4]
```diff
@@ -348,6 +361,8 @@ const CryptoDonation = (props: {
 					setFailedModalType={setFailedModalType}
 					setTxHash={setTxHash}
 					project={project}
+					mainProjectAddress={mainProjectAddress}
+					secondaryProjectAdress={secondaryProjectAddress}
```
Comment (MohammadPCh) [inline src/components/views/donate/CryptoDonation.tsx:365]: we never use these addresses in the donate modal. I think we can remove them.

File: src/components/modals/DonateModal.tsx [hunk 5]
```diff
@@ -34,6 +34,8 @@ export interface IDonateModalProps extends IModal {
 	anonymous?: boolean;
 	setSuccessDonation: (i: ISuccessDonation) => void;
 	givBackEligible?: boolean;
+	mainProjectAddress?: string;
+	secondaryProjectAdress?: string;
```
Comment (MohammadPCh) [inline src/components/modals/DonateModal.tsx:38]: we never use these addresses in the donate modal. I think we can remove them.

**Review by MohammadPCh — CHANGES_REQUESTED** [review]
Thank you @mateodaza 
It's great, just some small improvements suggested :)

File: src/config/development.ts (hunk 3 above)
Comment (mateodaza) [reply src/config/development.ts:13]: still on develop, gotta pay attention and coordinate with this [pr](https://github.com/Giveth/impact-graph/pull/531)

File: src/components/views/donate/CryptoDonation.tsx (hunk 4 above)
Comment (mateodaza) [reply src/components/views/donate/CryptoDonation.tsx:365]: we need it for the confirmDonation method :)

File: src/components/modals/DonateModal.tsx (hunk 5 above)
Comment (mateodaza) [reply src/components/modals/DonateModal.tsx:38]: same as above

**Review by MohammadPCh — APPROVED** [review]
Great Job, Thank you @mateodaza

**Pull request merged**